apps/dashboard/src/app/resources/edit/annotation/paragraph-annotation/paragraph-annotation.component.ts

```diff
@@ -114,6 +118,35 @@ export class ParagraphAnnotationComponent implements OnInit, OnDestroy {
     this.isModified = false;
   }
 
+  triggerSearch() {
+    this._triggerSearch(this.searchQuery).subscribe((results) => {
+      this.annotationService.setSearchResults(results);
+      this.updatePagination(results);
```

Comment:
The first search works properly. But If a second search is made with a different query, `page_number` is 1 instead of 0. Seems like `page_number` is not reset.

Reply:
good catch!